lightning-data-persister/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "lightning-data-persister"
+version = "0.0.1"
+authors = ["Valentine Wallace"]
```

Comment:
We don't have an explicit system atm. I was following Matt's lead I saw in #614 and just "taking the trophy," though I probably should've listed him as well because he wrote some of the original code.

No particular responsibilities for signing off on all changes afterward.

Reply:
Update: I decided to just put Matt as the author as a way of punting deciding on a system for this, lol.